app/controllers/user_auth_controller.rb

```diff
@@ -28,4 +28,8 @@ def user_not_authorized(exception)
     flash[:error] = t "#{policy_name}.#{exception.query}", scope: "pundit", default: :default
     redirect_to(request.referer || authenticated_user_root_path)
   end
+
+  def authenticated_user_root_path
+    current_user.only_invited? ? root_path : super
```

Comment:
Mmm,  c’est un peu bizarre de redéfinir une méthode définie par les routes. Est-ce qu’on peut supprimer cette route custom ? J’ai l’impression qu’en fait, c’est juste un alias de `users_rdvs_path`, et qu’on pourrait juste faire 

```suggestion
    current_user.only_invited? ? root_path : users_rdvs_path
```

ici.